CHANGELOG.md

```diff
@@ -22,7 +22,13 @@ Dropping a requirement of a major version of a dependency is a new contract.
 
 
 ## [Unreleased]
-[Unreleased]: https://github.com/atlassian-labs/db-replica/compare/release-2.0.0...master
+[Unreleased]: https://github.com/atlassian-labs/db-replica/compare/release-2.1.0...master
+
+### Fixed
+- Fix `Throttled Cache` lock release mechanism
```

Comment:
No mention about `LockBasedThrottledCache`

Reply:
It's not part of the API